README.md

```diff
@@ -1,57 +1,83 @@
+It is recommended to have a system that has at least 4+ GB memory and 32+ GB disk space with a dual core or better CPU. However, at a bare minimum having 2 GB memory, 8 GB disk space and a single core CPU should be enough to run this project.
 
-```
-npm run develop
-# or
-yarn develop
-```
+# 🚀 How to run locally?
 
-### `start`
+## Node
 
-Start your Strapi application with autoReload disabled. [Learn more](https://docs.strapi.io/developer-docs/latest/developer-resources/cli/CLI.html#strapi-start)
+This project uses Node. To be able to run this project on your local, you'll need to have `node` installed. You can check this by running `node --version` in your terminal. If you don't have it installed, you can follow the instructions [here](https://nodejs.org/en/download/).
 
-```
-npm run start
-# or
-yarn start
-```
+You will need to use at least version 14 of Node. Ideally, you should use LTS versions (i.e. 14, 16, 18, etc.) That is because Strapi does not support officially non-LTS versions, meaning there is a slight chance you might end up seeing some bugs.
+
+## Yarn
+
+To be able to run this project on your local, you'll need to have `yarn` installed. You can check this by running `yarn --version` in your terminal. If you don't have it installed, you can follow the instructions [here](https://classic.yarnpkg.com/en/docs/install/).
+
+## PostreSQL
+
+This project relies on having a PostgreSQL instance up and running. There are many ways of achieving this. You can read more at PostgreSQL's own website [here](https://www.postgresql.org/download/). If you're on a Mac, you can use [Postgres.app](https://postgresapp.com/). If you're on Windows, you can use [PostgreSQL for Windows](https://www.postgresql.org/download/windows/). If you're on Linux, you can use [PostgreSQL for Linux](https://www.postgresql.org/download/linux/). You will need your database information for the next step.
+
+To be able to run this project, you'll need to use at least version 11, but preferably version 14 of PostgreSQL.
+
+## Environment Variables
+
+This project relies on a `.env` file to supply the environment variables. For this, you'll need to create a `.env` file in the root directory of the project. The contents of the file should be as outlined in the `.env.example` file. You can copy the contents of the `.env.example` file into your `.env` file and fill in the values.
 
-### `build`
+Environment variables here are following closely what Strapi makes available, for which docs can be found at [Strapi's documentation](https://docs.strapi.io/dev-docs/configurations/environment).
 
-Build your admin panel. [Learn more](https://docs.strapi.io/developer-docs/latest/developer-resources/cli/CLI.html#strapi-build)
+## Running the project
 
+You are now set! You can run the following commands on your terminal to run the project:
+"develop": "strapi develop",
+"start": "strapi start",
+"build": "strapi build",
+"strapi": "strapi"
```

Comment:
Looking at `package.json`, you'll probably need to prefix those commands with `yarn`, no?